exercises/knapsack/description.md

```diff
@@ -0,0 +1,28 @@
+In this exercise, lets try to solve a classic problem.
+
+Bob is a thief. After months of planning, he 
+finally manages to crack the security systems of a 
+high-class apartment. In front of him are many 
+items, each with a value (v) and weight (w). Bob of
+course wants to maximise the total value he can 
+get; he would of course take all the items if he
+could. However, to his horrer, he realised that the 
```

Comment:
```suggestion
could. However, to his horror, he realized that the 
```

Reply:
It seems that I've suggested this correction once already.